src/GridList/GridTile.js

```diff
@@ -0,0 +1,142 @@
+// @flow weak
+
+import React, { Component } from 'react';
```

Comment:
`import React, { Component, Element } from 'react';`

...to support flow prop.